utils/network_protocol_pb2.py

```diff
@@ -278,6 +278,13 @@
       message_type=None, enum_type=None, containing_type=None,
       is_extension=False, extension_scope=None,
       serialized_options=None, file=DESCRIPTOR,  create_key=_descriptor._internal_create_key),
+    _descriptor.FieldDescriptor(
+      name='breath_id', full_name='SensorsProto.breath_id', index=5,
+      number=6, type=5, cpp_type=1, label=2,
+      has_default_value=False, default_value=0,
+      message_type=None, enum_type=None, containing_type=None,
+      is_extension=False, extension_scope=None,
+      serialized_options=None, file=DESCRIPTOR,  create_key=_descriptor._internal_create_key),
```

Comment:
Should I trust that this is generated by nanopb, and all info in there is correct?

Reply:
Yes, I regenerated everything using the commands in network_protocol.proto.